lib/widgets/vault_detail_button_stack.dart

```diff
@@ -144,6 +144,21 @@ class VaultDetailButtonStack extends ConsumerWidget {
                               ),
                             );
                           }
+
+                          // Delete Local Copy Button - shown after distribution when owner has a shard
+                          // Owner can delete vault content while keeping their shard for recovery
+                          if (backupConfig.lastRedistribution != null &&
+                              hasOwnerSteward(backupConfig) &&
```

Comment:
We should let the owner delete the contents even if they aren't holding a shard.